keyboards/zeix/singa/rubrehaku/keyboard.json

```
@@ -0,0 +1,129 @@
+{
+    "manufacturer": "KLC",
+    "keyboard_name": "Rubrehaku",
+    "maintainer": "itsme-zeix",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "rgblight": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true
+    },
+    "indicators": {
+        "caps_lock": "GP29"
+    },
+    "matrix_pins": {
+        "cols": ["GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1", "GP0"],
+        "rows": ["GP27", "GP28", "GP10", "GP11", "GP18", "GP19", "GP23", "GP24", "GP25", "GP26"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "static_light": true,
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "default": {
+            "val": 8,
+            "hue": 210,
+            "sat": 100,
+        },
+        "driver": "ws2812",
+        "led_count": 16,
+        "sleep": true,
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x8889",
+        "vid": "0x4C27"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Are the other layouts supported by the keyboard covered by this layout? If yes, `LAYOUT` should be used.
```suggestion
        "LAYOUT": {
```